pipelines/rancher/cluster.groovy

```diff
@@ -14,9 +14,9 @@ properties([
         jobsParameters.agents(),
         string(name: 'custom_cluster_name', defaultValue: '', description: 'Custom cluster name (Will override rancher_cluster_name)', trim: true),
         choice(name: 'eks_nodes_type', choices: ['SPOT', 'ON_DEMAND'], description: 'Select capacity associated with the EKS Node Group'),
-        string(name: 'asg_instance_types', defaultValue: 'm5.xlarge,m5a.xlarge,m5d.xlarge,m5ad.xlarge', description: 'List of EC2 shapes to be used in cluster provisioning', trim: true),
-        string(name: 'eks_min_size', defaultValue: '4', description: 'Minimum size of node group for eks cluster', trim: true),
-        string(name: 'eks_max_size', defaultValue: '8', description: 'Maximum size of node group for eks cluster', trim: true),
+        string(name: 'asg_instance_types', defaultValue: 'r5a.xlarge', description: 'List of EC2 shapes to be used in cluster provisioning', trim: true),
+        string(name: 'eks_min_size', defaultValue: '1', description: 'Minimum size of node group for eks cluster', trim: true),
```

Comment:
Maybe it would be better if the default minimum size of the cluster will correspond to the number of the availability zones

Reply:
yeah good catch